pkg/postgres/db_test.go

```diff
@@ -0,0 +1,118 @@
+package postgres
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	_ "github.com/DATA-DOG/go-sqlmock"
+	"github.com/bruin-data/bruin/pkg/query"
+	"github.com/jackc/pgx/v5/pgconn"
+	"github.com/pashagolub/pgxmock/v3"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClient_Select(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		query     string
+		expected  string
+		setupMock func(mock pgxmock.PgxPoolIface)
+		wantErr   string
+		want      [][]interface{}
+	}{
+		{
+			name:    "Test Select rows",
```

Comment:
nitpick, could you please make these all lowercase to be consistent with all the other tests?

Reply:
sure thing